core-graphics/src/event.rs

```diff
@@ -868,4 +874,5 @@ extern "C" {
 
     fn CGEventTapEnable(tap: CFMachPortRef, enable: bool);
 
+    fn CGEventSetLocation(event: crate::sys::CGEventRef, location: CGPoint);
```

Comment:
All other `CG...` functions here have rust documentation, so it makes sense to add it here as well, I think.

Reply:
Good point, I added to `CGEventTapEnable` as well.